src/Integration.Vsix/Telemetry/SonarLintIntegrationPackage.IOleCommandTarget.cs

```diff
@@ -1,77 +0,0 @@
-﻿//-----------------------------------------------------------------------
-// <copyright file="SonarLintIntegrationPackage.IOleCommandTarget.cs" company="SonarSource SA and Microsoft Corporation">
-//   Copyright (c) SonarSource SA and Microsoft Corporation.  All rights reserved.
-//   Licensed under the MIT License. See License.txt in the project root for license information.
-// </copyright>
-//-----------------------------------------------------------------------
-using Microsoft.VisualStudio;
-using Microsoft.VisualStudio.OLE.Interop;
-using Microsoft.VisualStudio.Shell;
-using System;
-using System.Diagnostics;
-using OLEConstants = Microsoft.VisualStudio.OLE.Interop.Constants;
-
-namespace SonarLint.VisualStudio.Integration.Vsix
-{
-    /// <summary>
-    /// Partial class to provide handling for SQM commands.
-    /// </summary>
-    [ProvideMenuResource("Menus.ctmenu", 1)]
-    partial class SonarLintIntegrationPackage : IOleCommandTarget
```

Comment:
All this moved to the MEF service almost as is (had to MEF import the service provider, but that is it).